lib/trino-parquet/src/main/java/io/trino/parquet/writer/PrimitiveColumnWriter.java

```diff
@@ -297,6 +297,7 @@ public long getBufferedBytes()
     public long getRetainedBytes()
```

Comment:
Is it called after a `flush`? (is there something like a flush here)?

Reply:
There is no "flush" here, there is a "close" followed by a `getBuffer` to extract the buffered pages.
This is called by connector page sink for every writer after writing each page.